pylsp/config/schema.json

```diff
@@ -49,6 +49,11 @@
       "default": false,
       "description": "Enable fuzzy when requesting autocomplete."
     },
+    "pylsp.plugins.jedi_completion.eager": {
```

Comment:
I don't know how we should manage these kind of configuration options that apply to both jedi, rope and possibly other completion backends

Reply:
I think splitting them by plugin is fine because some plugins may be very fast to eagerly resolve completions and some may be slow. It might be beneficial to let user/IDE make this choice (not necessarily today with rope and jedi as only completion providers, but potentially in the future).

This made me realise I missed adding the setting to schema for `rope`...